tests/monitors/marathon/marathon_test.py

```diff
@@ -31,3 +40,25 @@ def test_marathon(marathon_image):
 
             with run_agent(config) as [backend, _, _]:
                 assert wait_for(p(has_datapoint_with_dim, backend, "plugin", "marathon")), "didn't get datapoints"
+
+
+@pytest.mark.k8s
+@pytest.mark.kubernetes
+def test_marathon_in_k8s(agent_image, minikube, k8s_observer, k8s_test_timeout, k8s_namespace):
```

Comment:
Marathon is like a competitor to k8s and it doesn't really make sense to test it in k8s, so I would jsut remove this.